src/dbnode/storage/index_block_test.go

```diff
@@ -775,15 +814,6 @@ func TestLimits(t *testing.T) {
 			requireExhaustive: false,
 			expectedErr:       "",
 		},
-		{
-			name:              "no limits",
```

Comment:
why is this no longer relevant?

Reply:
it doesn't make sense that `exhaustive` would be false if there is no limit. just seems made up for the test.